fontbox/src/main/java/org/apache/fontbox/ttf/GlyphSubstitutionTable.java

```diff
@@ -75,6 +76,13 @@ public class GlyphSubstitutionTable extends TTFTable
 
     private GsubData gsubData;
 
+    /**
+     * The regex represents 4 'word characters' [a-zA-Z_0-9],
+     * see {@link java.util.regex.ASCII#WORD}<br>
+     * Note: the ' '-character is not matched!
+     */
+    static final Pattern wordPattern = Pattern.compile( "\\w{4}" );
```

Comment:
The name should be all uppercase for constants (static final). It should also be private.

Reply:
What do you think about making this a predicate?

```
    private static final Predicate<String> IS_FOUR_LETTERS = Pattern.compile( "\\w{4}" ).asMatchPredicate();
```

Then the test would be `IS_FOUR_LETTERS.test(featureTags[i])`.

Reply:
yeah, can be done for 3.0 and trunk.